src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-	  <VersionPrefix>11.7.2</VersionPrefix>
+	  <VersionPrefix>11.8.0</VersionPrefix>
 	  <!--For Feature packages update the VersionSuffix with the Jira ticket number-->
-	  <VersionSuffix>Alpha1</VersionSuffix>
+	  <VersionSuffix>Alpha-2</VersionSuffix>

```

Comment:
seems weird to increment the version number and pre-release version

Reply:
Currently this package is only available on my local machine. My plan was to remove the pre-release suffix after code review.

Reply:
Removed the suffix